source/manual/sentry.html.md

```diff
@@ -7,12 +7,215 @@ section: Monitoring
+[`active_sentry_environments`][active_sentry_environments] list, then none of
+its errors will be logged to Sentry.
+
+[active_sentry_environments]: https://github.com/alphagov/govuk_app_config/blob/b911c5bbef9bd1df6a92cf31eb5a8e0d3a91d851/lib/govuk_app_config/govuk_error/configure.rb#L14-L18
+
+### Advanced Sentry customisation
+
+For all of the above, you can easily
+[configure any of these properties by appending to them][error-configuration].
+
+You can also run arbitrary code to decide whether or not an error should be
+logged in Sentry, using the `should_capture` lambda. Your custom code will be
+[lazily combined with the default evaluators][combined-code] such that if an
+exception is on any of the 'excluded exceptions' lists, it will be excluded
+(even if your custom `should_capture` callback returns true).
+
+Find out more in the [govuk_app_config README][govuk_app_config-readme].
+
+[error-configuration]: https://github.com/alphagov/govuk_app_config/blob/master/README.md#error-configuration
+[combined-code]: https://github.com/alphagov/govuk_app_config/blob/b911c5bbef9bd1df6a92cf31eb5a8e0d3a91d851/lib/govuk_app_config/govuk_error/configuration.rb#L18-L24
+[govuk_app_config-readme]: https://github.com/alphagov/govuk_app_config/blob/master/README.md#error-reporting
+
+## When errors are received by Sentry
+
+Sentry first fingerprints the error to decide what Issue to group it under.
+It then checks how many occurrences of that Issue have happened recently,
+and rate-limits (i.e. ignores) this occurrence if it is happening too
+frequently. More details below.
+
+### Fingerprinting
+
+Errors are grouped into Issues automatically by Sentry. By default, Sentry
+[groups based on stack trace][stack-trace-grouping]. If no stack trace is
+available, it will [group by exception type][exception-grouping] instead.
+Failing that, it will group by error message.
+
+Fingerprint rules can be applied at the project level by editing the
+"Issue Grouping" page under a project (see [example][example-issue-grouping]).
+For example, you can force all errors of the same exception type to have the
+same fingerprint. On the same screen, you can set stack trace rules, to
+remove or rename certain 'frames' in the trace. In practice, we don't
+currently apply any custom rules, after an in-depth [investigation][]
+found that Sentry's default grouping was already very accurate (despite it
+often looking like the same exception is spread across multiple issues).
+
+Note that sentry-raven automatically cleans up stack traces for the purposes
+of fingerprinting, so that stack trace frames like:
+
+`app/views/welcome/view_error.html.erb in _app_views_welcome_view_error_html_erb__2807287320172182514_65600 at line 1`
+
+...get normalised to:
+
+`app/views/welcome/view_error.html.erb at line 1`
+
+This means if the same `ActionView::Template::Error` error happens twice, it
+is grouped into the same issue instead of erroneously treated as separate issues.
```

Comment:
I found this section a little hard to follow - I think this paragraph would make sense underneath the one below?

Reply:
Hmmm I've just tried this, but have moved it back because I think it makes more sense where it is. Have another read and let me know what you think. I've structured the section a bit better now so hopefully it is less confusing.

Reply:
Sure 👍 With the amendments I think this section generally reads better and so I'm resolving this comment, thanks!